packages/legacy/core/App/navigators/defaultStackOptions.tsx

```diff
@@ -4,7 +4,36 @@ import { useTranslation } from 'react-i18next'
 
 import HeaderTitle from '../components/texts/HeaderTitle'
 import { useConfiguration } from '../contexts/configuration'
-import { ITheme } from '../theme'
+import { ITheme, OnboardingTheme } from '../theme'
+import { ScreenOptionsType, Screens } from '../types/navigators'
+import { testIdWithKey } from '../utils/testable'
+
+const defaultScreenOptionsDictionary: ScreenOptionsType = {
+  [Screens.Onboarding]: {
+    headerTintColor: OnboardingTheme.headerTintColor,
+    headerLeft: () => false,
+  },
+  [Screens.Terms]: {
+    headerTintColor: OnboardingTheme.headerTintColor,
+    headerLeft: () => false,
+  },
+  [Screens.CreatePIN]: {
+    headerLeft: () => false,
+  },
+  [Screens.NameWallet]: {
+    headerTintColor: OnboardingTheme.headerTintColor,
+    headerLeft: () => false,
+  },
+  [Screens.UseBiometry]: {
+    headerTintColor: OnboardingTheme.headerTintColor,
+    headerLeft: () => false,
+  },
+  [Screens.Developer]: {
+    headerTintColor: OnboardingTheme.headerTintColor,
+    headerShown: false,
```

Comment:
Sorry I just caught this now, this will change the existing behavior. Could you remove this `headerShown: false`?

Reply:
Current behaviour is 
RootStack line 288 -> headerShown -> false
Then line 363, Screens.Developer -> no value to override the flag. That means default value for this screen is headerShown = false

Reply:
It's easy to miss but on line 288 `options={{ ...defaultStackOptions, title: t('Screens.Developer'), headerBackTestID: testIdWithKey('Back') }}` so that spreading of `defaultStackOptions` includes a `headerShown: true`